src/main/groovy/io/saagie/plugin/dataops/tasks/group/GroupListTask.groovy

```diff
@@ -17,10 +17,12 @@ class GroupListTask extends DefaultTask {
 	@Internal
 	SaagieClient saagieClient
 	
+	def result
```

Comment:
why sometimes you use `def` and sometimes you use `String`

Reply:
Good point, will fix it